### PR TITLE
#10913: Overload set 2 ops

### LINKED ISSUE #10913: Set 2 ops
Tracking Issue : #10833 
List of ops : 

- [x] NE
- [x] LE

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
Issue : #10913, Tracking Issue : #10833

https://github.com/tenstorrent/tt-metal/actions/runs/10200149733 - passed

### CODE REVIEW

File: ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward.hpp [hunk 1]
```diff
@@ -137,6 +137,21 @@ struct ExecuteBinaryBackwardBiasGelu {
 
 };
 
+struct ExecuteUnaryBackwardComparison {
```
Comment (ayerofieiev-tt) [inline ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward.hpp:140]: Does not look unary to me

File: ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp
Comment (ayerofieiev-tt) [inline ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp:1]: Please plan splitting this into .hpp and .cpp
Best example is ccl/all_gather

File: ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward.hpp (hunk 1 above)
Comment (ayerofieiev-tt) [reply ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward.hpp:140]: Should be renamed I think

**Review by ayerofieiev-tt — APPROVED** [review]
overall lgtm

**Review by eyonland — APPROVED** [review]
Please address @ayerofieiev-tt 's comments before merging to main.

File: ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward.hpp (hunk 1 above)
Comment (VirdhatchaniKN) [reply ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward.hpp:141]: We have renamed it in the next PR whoch will be merged along with this

Comment (VirdhatchaniKN) [reply ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward.hpp:141]: Sure

File: ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp
Comment (VirdhatchaniKN) [reply ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp:1]: We will work on this once after migration is done for sure

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/10211737127 - passed

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/10212438951 - passed

**Pull request merged**